airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -24,8 +24,8 @@ application {
 
 dependencies {
     implementation 'commons-codec:commons-codec:1.16.0'
-    implementation 'io.debezium:debezium-embedded:2.6.2.Final'
-    implementation 'io.debezium:debezium-connector-postgres:2.6.2.Final'
+    implementation 'io.debezium:debezium-embedded:3.0.1.Final'
+    implementation 'io.debezium:debezium-connector-postgres:3.0.1.Final'
```

Comment:
FYI Latest is 3.0.4
But I guess since this was tested and working we can leave at .1
I don't see any change that seems relevant in the following releases.

Reply:
thanks for this update! it's good to learn!